### PR TITLE
players: add developer column

### PR DESCRIPTION
**Pull request by laggron42**
So I added this new cool feature to see if someone is a developer or not

### CODE REVIEW

**Comment by MaxxMXSZ** [comment]
amazing✨

File: ballsdex/packages/players/cog.py [hunk 1]
```diff
@@ -90,3 +91,26 @@ async def donation_policy(
...
+    @app_commands.command()
+    async def developer(self, interaction: discord.Interaction, player: discord.User | None):
+        """
+        Check if a player is considered as a developer.
+
+        Parameters
+        ----------
+        player: discord.User | None
+            The player you want to inspect, if not yourself.
+        """
```
Comment (laggron42) [inline ballsdex/packages/players/cog.py:104]: Hmm, I don't really like the way this command is named.

File: ballsdex/packages/players/cog.py [hunk 2]
```diff
@@ -90,3 +91,26 @@ async def donation_policy(
...
+        user = player or interaction.user
+
+        try:
+            object = await PlayerModel.get(discord_id=user.id)
+        except DoesNotExist:
+            await interaction.response.send_message("The provided user is not a player.", ephemeral=True)
+            return
+        
+        if object.developer is True:
+            await interaction.response.send_message("Oh yeah that guy is a dev")
```
Comment (laggron42) [inline ballsdex/packages/players/cog.py:114]: I think this word fits better
```suggestion
            await interaction.response.send_message("Oh yeah that guy is a developer")
```

**Review by laggron42 — COMMENTED** [review]
Hmm, pretty nice, can you just change these two things

**Pull request closed without merging**